templates/config/70_nodepools.yaml.j2

```diff
@@ -0,0 +1,38 @@
+---
+nodepools:
+  abstract:
+    mech-ansible:
+      mechanism:
+        type: "ansible"
+        ansible:
+          topdir: "/home/duffy/src/duffy-ansible"
+          extra-vars:
+{% raw %}
+            nodepool: "{{ name }}"
+            template_name: "{{ name }}"
+{% endraw %}
+    virtual:
+      extends: "mech-ansible"
+      reuse-nodes: false
+      mechanism:
+        ansible:
+          extra-vars:
+            api_url: "http://n35.hufty.ci.centos.org:2633"
+            # api_username and api_password defined in 80_secrets.yaml
+          provision:
+            extra-vars:
+              # duffy_in.nodes gets passed in as a list
+              quantity: "{% raw %}{{ duffy_in.nodes | length }}{% endraw %}"
+            playbook: "provision.yml"
```

Comment:
all that should be variables and not making assumption about playbook to call ?

Reply:
`playbook` should probably be a variable, not sure about the `extra-vars` or `quantity`, respectively. We could just agree that the way to pass the number of hosts to be provisioned to be `quantity` and model our playbooks accordingly.